### PR TITLE
[FT]:[Growth 1032] limited gif resize

### PR DESCRIPTION
**Pull request by cyankhan**
**Issue #, if available:**

https://stocktwits.atlassian.net/browse/GROWTH-1032

**Description of changes:**

This limits resizing of gif to max its original dimensions

Test results : 
[all_pass.txt](https://github.com/stocktwits/serverless-image-handler/files/11652564/all_pass.txt)

**Checklist**
- [x] :wave: I have added unit tests for all code changes.
- [x] :wave: I have run the unit tests, and all unit tests have passed.
- [ ] :warning: This pull request might incur a breaking change.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: source/image-handler/index.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ export async function handler(event: ImageHandlerEvent): Promise<ImageHandlerExe
     };
   } catch (error) {
     console.error(error);
-
+    console.error("Error ocurred for path ", event.path);//log the path for debugging
```
Comment (Vinny49) [inline source/image-handler/index.ts:60]: "occurred"

File: source/image-handler/image-request.ts [hunk 2]
```diff
@@ -23,6 +24,12 @@ import https from 'https';
 import http from 'http';
 import { URL } from "url";
 
+const MAX_PERCENTAGE = parseInt(process.env.MAX_PERCENTAGE,10) || 75
+
```
Comment (Vinny49) [inline source/image-handler/image-request.ts:28]: I'd suggest tightening these lines up (no extra line break)

**Review by Vinny49 — APPROVED** [review]
Few small comments to address but good to go.

**Review by Vinny49 — APPROVED** [review]
Few small comments to address but good to go.

File: source/image-handler/index.ts (hunk 1 above)
Comment (cyankhan) [reply source/image-handler/index.ts]: updated

File: source/image-handler/image-request.ts (hunk 2 above)
Comment (cyankhan) [reply source/image-handler/image-request.ts]: updated

**Pull request merged**